src/lib/header-bidding/prebid/bid-config.ts

```diff
@@ -223,17 +224,35 @@ const openxClientSideBidder: (pageTargeting: PageTargeting) => PrebidBidder = (
 	},
 });
 
-const getOzonePlacementId = () => (isInUsa() ? '1420436308' : '0420420500');
+const getOzonePlacementId = (sizes: HeaderBiddingSize[]) => {
+	if (isInUsa()) {
+		if (getBreakpointKey() === 'D') {
+			if (containsBillboard(sizes)) {
+				return '3500010912';
+			}
+
+			if (containsMpu(sizes)) {
+				return '3500010911';
+			}
+		}
+		return '1420436308';
+	} else {
+		return '0420420500';
+	}
```

Comment:
Do we need this else? If `0420420500` is a default option we could just return it?

Reply:
Good point, this rule might be handy! https://eslint.org/docs/latest/rules/no-else-return